pkg/cmd/images/images.go

```diff
@@ -0,0 +1,407 @@
+/*
```

Comment:
Probably this should be moved to `pkg/cmd/image/list.go`.
`pkg/cmd/load` should be merged into `pkg/cmd/image` too. (Can be another PR)